fontforge/parsettf.c

```diff
@@ -2796,6 +2796,10 @@ return( 3 );
 	pt = buffer;
 	do {
 	    ch = getc(ttf);
+		if ((pt-buffer) > (sizeof(buffer) - 3)) {
```

Comment:
Hi @jtanx, the rest of the CVE-related commits LGTM, but here I would add a comment about what does it mean that `(pt-buffer) > (sizeof(buffer) - 3)`.